pom.xml

```diff
@@ -1732,6 +1732,7 @@
                                     readable than the alternative. -->
                                     -Xep:YodaCondition:OFF
                                     -XepOpt:CheckReturnValue:CheckAllConstructors=true
+                                    -XepOpt:CanonicalConstantFieldName:ExcludedConstantFliedNames=serialVersionUID
```

Comment:
Alternative is to follow the convention for this variable obviously 😅

Reply:
I think this should be a default exclusion; otherwise _all_ users will have to make this change (`serialVersionUID` cannot be renamed: Java requires it to be spelled exactly like this).